### ISSUE TITLE
Fix href accessibility issues 

### ISSUE THREAD

**Issue by jdoiron94** [jdoiron94/questly#16]
The error message close button is currently an  `a` tag with an invalid href. This causes it to appear as an interactable button that calls a function without the traditional button styling. This is an accessibility issue and should be corrected.

The purpose of this ticket is to change the element to a `button` tag and control the styles on it so it renders as if it were just a link.
